README

```diff
@@ -236,8 +236,11 @@ MellonPostCount 100
         # set using MellonSetEnv into single variable:
         # ie: MYENV_VAR => val1;val2;val3 instead of default behaviour of:
         #     MYENV_VAR_0 => val1, MYENV_VAR_1 => val2 ... etc.
+        # Second optional parameter specifies the separator, to override the
+        # default semicolon.
         # Default: MellonMergeEnvVars Off
         MellonMergeEnvVars On
+        MellonMergeEnvVars On :
```

Comment:
Hi, I think that for everybody's sanity, we should encourage them to enclose the second parameter in doublequotes. I.e.:

```
MellonMergeEnvVars On ":"
```